### DISCUSSION TITLE
Remove 2FA in Setting if Login with Proton.me account?

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#1861] [Feature Requests]
I was tripping myself up mentally when I was able to create an additional 2FA one-time passcode whilst being logged in/verfied via my Proton account (that used it's own one-time associated with Proton). 

This SimpleLogin one-time would never be used to loging to anything but would just keep running in the background. I think it is very odd that this is possilbe and perhaps this should be removed? Perhaps this is already on your list to improve with further integrations with Proton but thought I'd mention.

Cheers!

**Comment by nguyenkims** [comment]
Thanks for the report. A person can decide to set a password for their account even if they use "login with proton" and in this case having 2FA can be useful.

**Reply by ghost** [reply]
Maybe I'm missing something but how would it be useful as it would never have the ability to be used?      So one can login with Proton and separatly with their own login? Not connecting the dots.
